Donation-platform/src/main/java/com/google/charities/CharityClone.java

```diff
@@ -0,0 +1,84 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.charities;
+
+import java.util.Collection;
+
+/** Clone of a Charity to store data from charity JSONs: name, link, category, description, rating. */
+public final class CharityClone {
```

Comment:
please rename this class. why do we need this clone class? why not use existing charity class.

Reply:
I replied to Toni with my reasoning above. If it doesn't make sense / would be worth changing my approach to JSON parsing, please let me know and I can further explain and/or make those changes.

Reply:
CharityClone is removed. I am now using a HashMap to transfer the values from the JSON into the addCharity constructor.